frontend/templates/lobby.html

```diff
@@ -1610,6 +1610,31 @@
 
             return circleData;
         }
+        
+        let sizeSlot = 8;
+        function wheelSize(e) {
+            event.preventDefault();
+            sizeSlot += Math.min(Math.max(-1, e.deltaY), 1);
+            if (sizeSlot < 6) {
+                sizeSlot = 6;
+            } else if (sizeSlot > 32) {
+                sizeSlot = 32;
+            }
+
+            if(sizeSlot < 12)
+                n = 8
```

Comment:
Is n even necessary? Could just use `sizeSlot` or furthermore `lineWidthUnscaled`